tools/build_api.py

```diff
@@ -191,8 +191,8 @@ def is_official_target(target_name, version):
     if hasattr(target, 'release_versions') \
        and version in target.release_versions:
         if version == '2':
-            # For version 2, either ARM or uARM toolchain support is required
-            required_toolchains = set(['ARM', 'uARM'])
+            # For version 2, one of the ARM toolchains(ARM, ARMC6, ARMC5 or uARM) support is required
+            required_toolchains = set(['ARM', 'ARMC5', 'ARMC6', 'uARM'])
```

Comment:
ARM and uARM is mandatory for Mbed 2. But ARMC5 and ARMC6 are not. 
```suggestion
           required_toolchains = [
                set(['ARM', 'uARM']),
                set(['ARMC5', 'uARM'])
            ]
```

Reply:
I thought its other way, uARM is supported for Mbed2, but not a must/required for Mbed2.
Please see the next line, the intention is to check for if one of ARM toolchains are supported.

Reply:
Sorry I got confused with 5.